sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -28,319 +34,1077 @@ public CertificateClient(Uri vaultUri, TokenCredential credential)
+            scope.Start();
+
+            try
+            {
+                Response<ContactList> contactList = await _pipeline.SendRequestAsync(RequestMethod.Get, () => new ContactList(), cancellationToken, ContactsPath).ConfigureAwait(false);
+
+                return new Response<IList<Contact>>(contactList.GetRawResponse(), contactList.Value.ToList());
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
         }
 
-        public virtual IAsyncEnumerable<Response<Contact>> GetContactsAsync(CancellationToken cancellationToken = default)
+        public virtual Response<IList<Contact>> DeleteContacts(CancellationToken cancellationToken = default)
         {
-            throw new NotImplementedException();
+            using DiagnosticScope scope = _pipeline.CreateScope("Azure.Security.KeyVault.Certificates.CertificateClient.DeleteContacts");
+            scope.Start();
+
+            try
+            {
+                Response<ContactList> contactList = _pipeline.SendRequest(RequestMethod.Delete, () => new ContactList(), cancellationToken, ContactsPath);
+
+                return new Response<IList<Contact>>(contactList.GetRawResponse(), contactList.Value.ToList());
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
         }
 
-        public virtual IEnumerable<Response<Contact>> DeleteContacts(CancellationToken cancellationToken = default)
+        public virtual async Task<Response<IList<Contact>>> DeleteContactsAsync(CancellationToken cancellationToken = default)
         {
-            throw new NotImplementedException();
+            using DiagnosticScope scope = _pipeline.CreateScope("Azure.Security.KeyVault.Certificates.CertificateClient.DeleteContacts");
+            scope.Start();
+
+            try
+            {
+                Response<ContactList> contactList = await _pipeline.SendRequestAsync(RequestMethod.Delete, () => new ContactList(), cancellationToken, ContactsPath).ConfigureAwait(false);
+
+                return new Response<IList<Contact>>(contactList.GetRawResponse(), contactList.Value.ToList());
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
         }
 
-        public virtual IAsyncEnumerable<Response<Contact>> DeleteContactsAsync(CancellationToken cancellationToken = default)
+        internal CertificatePolicy CreateDefaultPolicy()
         {
-            throw new NotImplementedException();
+            var policy = new CertificatePolicy
+            {
+                IssuerName = "Self",
+                Subject = "CN=default",
+                KeyOptions = new RsaKeyOptions(false)
+                {
+                    Exportable = true,
+                    ReuseKey = false
```

Comment:
Nit: recommend terminating comma separators. Makes future diffs/merges easier to read/do.